asn1c_combined/FixIt.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+asn1c -Wdebug-compiler -fcompound-names -gen-PER -pdu=all 1609dot2-base-types.asn 1609dot2-schema.asn J2735_201603DA.ASN 2>&1 | tee compile.out
+
+sed -i 's/\(-DASN_PDU_COLLECTION\)/-DPDU=MessageFrame \1/' Makefile.am.example
+
+sed -i 's/#include "Latitude.h"/#include "DSRC_Latitude.h"/' *DLocation.h
```

Comment:
There is a pull request for the fix to issue #211 that you reported. If these changes are to work around that, you may want to try the PR to see if these are still necessary.